src/aura/BGE_Wizard/BGE_WizardHelper.js

```diff
@@ -764,13 +834,61 @@
                 this.onMetadataUpdated.notify();
             }
 
+            /**
+             * @description sets the page header based on the step in the wizard
+             * @return void.
+             */
+            function setPageHeader() {
+                switch (parseInt(this.progressIndicatorStep)) {
+                    case 1:
+                        this.pageHeader = $A.get('$Label.c.bgeBatchTemplateOverviewWizard');
```

Comment:
Could we move step configuration into an array so we can maintain simple increment/decrement logic?

```
If it's just headers ... 
var headers = [ 
   $A.get('$Label.c.bgeBatchTemplateOverviewWizard'), 
    'Select Template',
    ...
]
```

or if there's other data, move it into a step object?